hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java

```diff
@@ -0,0 +1,489 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
```

Comment:
@JeffreyDallas  Thank you ! So, after this will JRS tests be validating the V6 record stream files ?

Reply:
Never mind, saw this PR https://github.com/swirlds/swirlds-platform-regression/pull/2997/files